src/constants.ts

```diff
@@ -0,0 +1,15 @@
+export const JWT_ISSUER = "carrot-data-uploader";
+export const NONCE_LENGTH_BYTES = 32;
+
+export enum DataFormat {
+    Json = "json",
+}
+
+export const MIME_TYPE: Record<DataFormat, string> = {
+    [DataFormat.Json]: "application/json",
+};
+
+export enum StorageService {
+    Ipfs = "ipfs",
```

Comment:
Should we rename this to `w3up` to just be more consistent?

Reply:
I think this is fine as we're not storing on `w3up`. W3up is the API that we use to store and persist data to IPFS, so it's ultimately IPFS we're storing on. As for the other `w3up` references, in those cases we're interacting with the `w3up` API, so I think it's right that is says `w3up` and not IPFS. This makes it easier to switch the IPFS pinning/persistance service in the future if we want to.